profiles/repositories.py

```diff
@@ -68,8 +68,9 @@ def get(self, orcid: str) -> OrcidToken:
         try:
             return self.db.session.query(OrcidToken).filter_by(orcid=orcid).one()
         except NoResultFound as exception:
-            raise OrcidTokenNotFound('ORCID token for the ORCID {} not found'.format(orcid)) \
-                from exception
+            msg = 'ORCID token for ORCID {} not found'.format(orcid)
+            LOGGER.info(msg=msg)
```

Comment:
Just wondering, whether logging exception messages adds much. Shouldn't it be logged anyway, where the exception is handled?

Reply:
I thought so but couldn't find them when we needed, so erring on the side of caution

Reply:
I also do not see these exceptions when looking through the logs. I suspect that they are more for generating HTTP responses.

Reply:
Maybe the logging could be hooked in at a higher level to catch more errors?

Reply:
https://rpm.newrelic.com/accounts/1451451/applications/63869969/traced_errors